src/library.js

```diff
@@ -302,13 +302,28 @@ LibraryManager.library = {
   },
   getpagesize: function() {
     // int getpagesize(void);
+#if MINIMAL_RUNTIME // MINIMAL_RUNTIME does not have a global PAGE_SIZE runtime variable.
+#if WASM
+    return {{{ WASM_PAGE_SIZE }}};
+#else
+    return {{{ ASMJS_PAGE_SIZE }}};
+#endif
+#else
     return PAGE_SIZE;
```

Comment:
How about leaving the minimal runtime code path here, for both runtimes? There's no benefit I can see to having both (basically, we can avoid using the global runtime variable in all cases, here).

Reply:
That sounds good to me, but can that break some users that expect to be able to reference `PAGE_SIZE` variable in their library/--pre-js code that references the classic runtime? Perhaps that can be handled as a separate change?

Reply:
Sorry if I wasn't clear - I'm not suggesting removing the `PAGE_SIZE` variable wherever it is declared for the normal runtime. I'm suggesting in this location here to not use that variable, and use the same code as the minimal runtime. It avoids an ifdef on the runtime, basically.

Reply:
Ok, I applied the change only for this location. I think that follows what you meant. Does that work?

Reply:
lgtm with this change.